Dockerfile.jbvc

```diff
@@ -13,6 +13,14 @@ RUN conda env create -f /VariantCalling/setup/environment.yml && \
     conda env create -f /VariantCalling/setup/other_envs/cnmops.yml && \
     conda init --all
 
+## install ug-cn.mops:
+RUN cd /opt && git clone --branch ilyasoifer/test.1.40 https://github.com/Ultimagen/cn.mops.git cn.mops
+
+RUN source ~/.bashrc && \
+   conda activate cn.mops && \
+   Rscript -e 'remove.packages("cn.mops")' && \
```

Comment:
Cleaner to remove cn.mops from the environment I think

Reply:
I don't agree. 
the conda environment should include some version of cn.mops. 
We decide to override the installation with another/dev version.  
I am not sure, but maybe we can export the conda environment after installation...?